packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.11.0
+
+* **Breaking Change** Changes the Android implementation of the camera plugin from `camera_android`
```

Comment:
I think we should probably say something positive here about CameraX, so it doesn't sound like this is a step backward. E.g., something like "... to `camera_android_camerax`, which has better support for a wider range of devices. The CameraX implementation has feature parity with `camera_android` except for ..."